<launch>

    <node pkg="ti_viz_nodes" type="viz_objdet" name="viz_objdet" output="screen" args="" required="true">

        <!-- Input recitified image topic name to subscribe to -->
        <param name="rectified_image_topic"    value="camera/right/image_rect_rgb"/>

        <!-- Input CNN tensor topic name to subscribe to -->
        <param name="vision_cnn_tensor_topic"  value="vision_cnn/tensor"/>

        <!-- Output semantic segmentation image topic name to publish to -->
        <param name="vision_cnn_image_topic"   value="vision_cnn/out_image"/>

    </node>

</launch>
